app/services/return-logs/generate-return-log.service.js

```diff
@@ -5,6 +5,7 @@
  * @module GenerateReturnLogService
  */
 
+const FeatureFlagsConfig = require('../../../config/feature-flags.config.js')
```

Comment:
This needs to go below the other `requires()`. See [Top of .js files](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files)

```suggestion
const featureFlagsConfig = require('../../../config/feature-flags.config.js')
```